src/main/java/org/traccar/protocol/TranSyncProtocol.java

```diff
@@ -0,0 +1,22 @@
+package org.traccar.protocol;
+
+import org.traccar.BaseProtocol;
+import org.traccar.PipelineBuilder;
+import org.traccar.TrackerServer;
+import org.traccar.config.Config;
+
+import javax.inject.Inject;
+
+public class TranSyncProtocol extends BaseProtocol {
+
+    @Inject
+    public TranSyncProtocol(Config config) {
+        addServer(new TrackerServer(config, getName(), false) {
+            @Override
+            protected void addProtocolHandlers(PipelineBuilder pipeline, Config config) {
+                pipeline.addLast(new TranSyncProtocolDecoder(TranSyncProtocol.this));
```

Comment:
Frame decoder seems to be missing.

Reply:
If LengthFieldBasedFrameDecoder for FrameDecoder used, should I implement sendResponse function?

I cant`t find our AIS 140  device to make tests.

Reply:
How is response related to frame decoder? I'm confused.